Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -1620,5 +1638,56 @@ - (void)downloadProgressDidUpdateTo:(double)progress forBook:(NYPLBook * _Nonnul
 }
 #endif
 
+#if FEATURE_AUDIOBOOKS
+/// Create an audiobook object and add to download queue
+- (void)downloadAudiobookForBook:(NYPLBook *)book {
+  [AudiobookManifestAdapter transformAudiobookManifestWithBook:book
```

Comment:
can `book` be nil?

Reply:
Guard statement is added in the `AudiobookManifestAdapter` method.